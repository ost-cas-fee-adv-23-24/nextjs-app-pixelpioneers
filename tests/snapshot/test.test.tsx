import React from 'react';
import { Button } from '@ost-cas-fee-adv-23-24/design-system-pixelpioneers';
import { render } from '@testing-library/react';

describe('Home', () => {
    it('renders a heading', () => {
        const button = render(<Button label="hey" />);

        // expect('hey').toBeInTheDocument();

        expect(button).toMatchSnapshot();
    });
});
