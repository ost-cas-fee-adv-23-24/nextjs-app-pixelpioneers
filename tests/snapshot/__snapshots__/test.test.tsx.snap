// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders a heading 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <button
        class="flex h-fit justify-center w-fit hover:ring-3 active:ring-4 bg-primary-600 fill-white text-white hover:bg-primary-700 hover:ring-primary-100 active:bg-primary-700 active:ring-primary-200 gap-xs rounded-s p-base transition-all duration-350 ease-in-out active:duration-300 disabled:bg-secondary-400 disabled:cursor-not-allowed"
      >
        <span
          class="wrap-label font-default font-semibold leading-none text-sm text-inherit"
        >
          Login with Zitadel
        </span>
        <svg
          aria-hidden="true"
          class="self-center"
          height="16"
          viewBox="0 0 16 16"
          width="16"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M8 7a3.5 3.5 0 1 0 0-7 3.5 3.5 0 0 0 0 7ZM8 8.5a7.008 7.008 0 0 0-7 7 .5.5 0 0 0 .5.5h13a.5.5 0 0 0 .5-.5 7.008 7.008 0 0 0-7-7Z"
          />
        </svg>
      </button>
    </div>
  </body>,
  "container": <div>
    <button
      class="flex h-fit justify-center w-fit hover:ring-3 active:ring-4 bg-primary-600 fill-white text-white hover:bg-primary-700 hover:ring-primary-100 active:bg-primary-700 active:ring-primary-200 gap-xs rounded-s p-base transition-all duration-350 ease-in-out active:duration-300 disabled:bg-secondary-400 disabled:cursor-not-allowed"
    >
      <span
        class="wrap-label font-default font-semibold leading-none text-sm text-inherit"
      >
        Login with Zitadel
      </span>
      <svg
        aria-hidden="true"
        class="self-center"
        height="16"
        viewBox="0 0 16 16"
        width="16"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M8 7a3.5 3.5 0 1 0 0-7 3.5 3.5 0 0 0 0 7ZM8 8.5a7.008 7.008 0 0 0-7 7 .5.5 0 0 0 .5.5h13a.5.5 0 0 0 .5-.5 7.008 7.008 0 0 0-7-7Z"
        />
      </svg>
    </button>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
